README.md

```diff
@@ -238,9 +238,7 @@ turned ON or OFF through CMake variables.
 
 All dependencies are supplied by the Fletch package of 3rd party dependencies.
 
-[Eigen](http://eigen.tuxfamily.org/) (>= 3.0)
-[log4cxx] (https://logging.apache.org/log4cxx/) (>= 0.10.0)
-[Apache Runtime] (https://apr.apache.org/)
+[Eigen](http://eigen.tuxfamily.org/) (>= 3.0) [log4cxx](https://logging.apache.org/log4cxx/) (>= 0.10.0) [Apache Runtime](https://apr.apache.org/)
```

Comment:
No, that wasn't intentional I just wanted to fix the way the links displayed with Github's markdown parsing. Multiple lines makes sense to me.